roles/powervs_client_enable_services/tasks/main.yml

```diff
@@ -69,7 +77,7 @@
         name: chrony
         state: present
 
-    ## modifying /etc/chronyd file
+    ## Modifying /etc/chronyd file
     - name: Replace iburst line in /etc/chronyd file
```

Comment:
Replace iburst line in /etc/chrony.conf file

Reply:
Corrected